packages/hub/card-document.ts

```diff
@@ -44,6 +45,9 @@ export class CardDocument {
               this.csOriginalRealm = value;
             }
             break;
+          case 'csCreatedBy':
+            this.csCreatedBy = value;
+            break;
```

Comment:
this is incorrect, instead of this just add `csCreatedBy` to the `case` list below.